devicon.json

```diff
@@ -2525,6 +2525,27 @@
           }
         ]
     },
+    {
+        "name": "sqlalchemy",
+        "tags": [
+            "python",
+            "orm"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark",
+                "plain"
+            ],
+            "font": [
+                "original",
+                "original-wordmark",
+                "plain"
+            ]
```

Comment:
I would suggest to have `plain` and `plain-wordmark` as font versions and to move the original over to the `aliases` section (https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md#-updating-the-deviconjson-). This result in the advantage to not have the same icon twice in the font file (since the color is removed when beeing transformed to font, see the `bot:peek` screenshots).